module/Decision/src/Decision/Controller/MemberController.php

```diff
@@ -19,6 +20,23 @@ public function indexAction()
     }
 
     /**
+     * Search action, allows searching for members.
+     */
+    public function searchAction()
+    {
+        $name = $this->params()->fromRoute('name');
+        $members = array();
+        foreach ($this->getMemberService()->findMemberByName($name) as $member) {
+            //TODO: this returns a lot of data, much more than is needed in most cases.
```

Comment:
By aan lot of data I meant that toArray() returns a lot of data such as email, expiration, generation, etc. although limiting the number of results still is a good idea.